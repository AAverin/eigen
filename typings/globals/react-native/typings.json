{
  "resolution": "main",
  "tree": {
    "src": "https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/4c7d03b088d4e71cd9fe9c66ab7bd363049077c3/react-native/react-native.d.ts",
    "raw": "registry:dt/react-native#0.29.0+20160719232534",
    "typings": "https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/4c7d03b088d4e71cd9fe9c66ab7bd363049077c3/react-native/react-native.d.ts"
  }
}
