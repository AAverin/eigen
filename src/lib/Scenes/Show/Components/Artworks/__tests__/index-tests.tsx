import GenericGrid from "lib/Components/ArtworkGrids/GenericGrid"
import React from "react"
import "react-native"
import * as renderer from "react-test-renderer"
import { ShallowRenderer } from "react-test-renderer/shallow"
import { Artworks } from "../index"

it("looks correct when rendered", () => {
  const artwork = [{ id: "12345" }]
  const comp = renderer.create(
    <Artworks>
      <GenericGrid artworks={artwork} />
    </Artworks>
  )
  expect(comp).toMatchSnapshot()
})
