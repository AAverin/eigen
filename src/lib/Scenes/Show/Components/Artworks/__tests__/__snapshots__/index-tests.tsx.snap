// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`looks correct when rendered 1`] = `
Array [
  <Text
    accessible={true}
    allowFontScaling={true}
    className="sc-bwzfXH uTxCW"
    ellipsizeMode="tail"
    fontFamily="AGaramondPro-Regular"
    fontSize={22}
    lineHeight={32}
    my={1}
    style={
      Array [
        Object {
          "fontFamily": "AGaramondPro-Regular",
          "fontSize": 22,
          "lineHeight": 32,
          "marginBottom": 4,
          "marginTop": 4,
        },
        undefined,
      ]
    }
  >
    All works
  </Text>,
  <View
    onLayout={[Function]}
  >
    <View
      accessibilityLabel="Artworks Content View"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    />
  </View>,
  <Text
    accessible={true}
    allowFontScaling={true}
    className="sc-bdVaJa bDWFJH"
    ellipsizeMode="tail"
    fontFamily="Unica77LL-Medium"
    fontSize={14}
    lineHeight={24}
    my={2}
    style={
      Array [
        Object {
          "fontFamily": "Unica77LL-Medium",
          "fontSize": 14,
          "lineHeight": 24,
          "marginBottom": 8,
          "marginTop": 8,
        },
        undefined,
      ]
    }
  >
    View all works
  </Text>,
]
`;
